src/frontend/README.md

```diff
@@ -10,6 +10,21 @@ Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app w
 
 Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|module`.
 
+## Installing yarn
+
+To install yarn follow the instruction provided on https://yarnpkg.com/lang/en/docs/install/#windows-tab
+
+To enable Yarn for Angular CLI, run the following command inside root directory: 
+
+`ng set packageManager=yarn`
+
+To install dependencies run the following command inside root directory:
+`yarn`  
+
+#### Why Yarn?
+* Yarn parallelizes operations to maximize resource utilization so install times are faster than ever 
+For more detail information read https://yarnpkg.com/en/
```

Comment:
thanks but need more generic info see https://medium.com/@nikjohn/facebooks-yarn-vs-npm-is-yarn-really-better-1890b3ea6515 and pick some verified info for bullet points

Also mention, how we  (a developer) can add /remove  package  and save to yarn.lock